### DISCUSSION TITLE
logfile for automad

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#45] [General]
hi
is there somewhere a logfile in automad?
Since about 3 nights I got every morning same message in automad:
![grafik](https://github.com/user-attachments/assets/1a68409b-142f-4433-9eac-f5add8733c29)

After a look inside the webhosting I can see there is a new folder named .cagefs. Created every night at 0337. As soon I delete it, the message is gone.
Hoster said this is a problem from automad tries to write outside the main folder called httpdocs, wherin I installed automad. But I can't believe that...

Because deleting the folder every morning cannot be the solution, I'm asking here ;-)

Any answer is greatly appreciated. thx

**Comment by marcantondahmen** [comment]
I think the best would be to change hoster. The answer the gave you is simply a joke. It looks like the directory is created by them every night and contains a link to a location that can't be accessed by PHP. This is can't be real. 

The real question is why do they put this nonsense directory there? To me it looks like a bad setup.

I will take look at it later. Maybe there is an easy solution.

**Comment by AtjonTV** [comment]
Lol. Just found this thread and looked up "cagefs": https://support.cpanel.net/hc/en-us/articles/1500009717042-What-is-CageFS

This is 100% a hoster issue. And because of them telling you its automad's fault, makes me think the hoster doesnt even understand the software they are using (cPanel).

**Comment by cracksilver** [comment]
That’s what I thought as well ;-)
So I changed the hoster to get rid of this issue.
